### DISCUSSION TITLE
Happy Anniversary to rAudio

### DISCUSSION THREAD

**Discussion by EarsofTin** [rern/rAudio#697] [General]
I believe rAudio turns one year old today. Thank you Rern for all your efforts to make the this the best!

**Comment by ceejayemm** [comment]
Well done @rern, a great decision to move away from the old RuneAudio web site and branch out on your own.  The product gets better and better.

Happy Christmas (Holidays - which ever is your way of celebrating). Keep it up in the New Year.

Chris

**Comment by Raitsa** [comment]
Very happy with rAudio. Got it running in 2 rpis at the moment, 2 more coming up.

Fresh review and feedback here:
https://rawdlite-audio-doc.readthedocs.io/en/latest/distributions.html

**Reply by rern** [reply]
Have you try **Audiophonics I-Sabre DAC ES9038Q2M** for your Evo-Sabre?

**Reply by rawdlite** [reply]
Maybe I can answer this one.
There is some code maintained by audiophonics to build custom images giving access to various added features for the Evo Sabre Dac/Streamer.
https://github.com/audiophonics/Evo_Sabre/tree/main
images are for volumio and moode only. Maybe an adaption for rAudio is feasible?
BTW the text linked to is not finalized and more of an exercise in reStructuredText at this moment.
BTW2 would it be feasible to exchange mpd for mopidy?
BTW3 Congratulations on the anniversary - great work.

**Comment by cYCL157** [comment]
Happy anniversary!  I'd be lost without rAudio, it's part of my daily life!
